helix-term/src/commands/typed.rs

```diff
@@ -459,10 +459,14 @@ fn format(
     if let Some(format) = doc.format() {
         let callback = make_format_callback(doc.id(), doc.version(), view.id, format, None);
         cx.jobs.callback(callback);
+    } else {
+        cx.editor
+            .set_error("Formater not available and no LSP has formatting capabilities");
```

Comment:
```suggestion
            .set_error("The formatter is not available, and no language server offers formatting capabilities");
```
Formatter is the correct spelling, and LSP is an acronym for Language Server Protocol, this refers to the language server itself. I see these terms seem to be used interchangeably in the documentation at the moment, so @the-mikedavis what do you think?

Reply:
Yeah good point, using "LSP" to mean "language server" is a minor nit of mine. I'll push a change to eliminate the improper "LSP" usage.

Reply:
I actually like `language server` (I think avoiding acronyms where possible is a good rule of thumb for users), so updated the PR to include the new wording of the error.

Thank you for the fixes in the general wording of the error.